subsys/net/l2/ethernet/gptp/gptp_state.h

```diff
@@ -374,6 +386,27 @@ struct gptp_clk_slave_sync_state {
 	bool rcvd_local_clk_tick;
 };
 
+/* ClockMasterSyncOffset state machine variables. */
+struct gptp_clk_master_sync_offset_state {
+	/** Current state of the state machine. */
+	enum gptp_cms_offset_states state;
+
+	/** Notifies the state machine when Sync Receipt Time is received. */
+	bool rcvd_sync_receipt_time;
+};
+
+/* ClockMasterSyncSend state machine variables. */
+struct gptp_clk_master_sync_snd_state {
+	/** Current state of the state machine. */
```

Comment:
Shuffle the struct fields so that there are no holes, for example the enum should be last here.

Reply:
Done